content/en/docs/reference/command-line-tools-reference/feature-gates/RecoverVolumeExpansionFailure.md

```diff
@@ -13,6 +13,9 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.32"
+  - stage: stable
```

Comment:
- stage: beta
    defaultValue: true
    fromVersion: "1.32"
    toVersion: "1.33"

Reply:
@gnufied Can you address the comment?